examples/app/values.yaml

```diff
@@ -76,6 +76,10 @@ myapp:
   nodeSelector:
     region: east
     type: user-node
+  podSecurityContext:
+    fsGroup: 20000
+    runAsNonRoot: true
+    runAsUser: 65532
```

Comment:
I can see that that previous value in template was:
```yaml
securityContext:
        runAsNonRoot: true
```
where `fsGroup` and `runAsUser` comes from?

Reply:
if these values were added manually, they will be removed on next "generate examples" run. Please add these fields to source manifests `test_data/sample-app.yaml`

Reply:
Sorry, forgot to add test_data changes to commit.
Will do right now.

Reply:
Is it ok now? @arttor